app/ui-react/packages/ui/src/Data/Virtualizations/Views/ViewHeaderBreadcrumb.tsx

```diff
@@ -0,0 +1,232 @@
+import * as H from '@syndesis/history';
+import {
+  DropdownKebab,
+  MenuItem,
+  // OverlayTrigger,
+  // Tooltip,
+} from 'patternfly-react';
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { Breadcrumb, ButtonLink } from '../../../Layout';
+
+import {
+  ConfirmationButtonStyle,
+  ConfirmationDialog,
+  ConfirmationIconType,
+} from '../../../Shared';
+
+import {
+  BUILDING,
+  CONFIGURING,
+  DEPLOYING,
+  RUNNING,
+  SUBMITTED,
+  VirtualizationPublishState,
+} from '../models';
+
+import './ViewHeaderBreadcrumb.css';
+
+export interface IViewHeaderBreadcrumbProps {
+  currentPublishedState: VirtualizationPublishState;
+  dashboardHref: H.LocationDescriptor;
+  dataHref: H.LocationDescriptor;
+  dashboardString: string;
+  dataString: string;
+  i18nViews: string;
+  i18nCancelText: string;
+  i18nDelete: string;
+  i18nDeleteModalMessage: string;
+  i18nDeleteModalTitle: string;
+  // i18nDraft: string;
+  // i18nError: string;
+  // i18nEdit: string;
+  // i18nEditTip?: string;
+  /* TD-636: Commented out for TP
+  i18nExport: string; */
+  i18nPublish: string;
+  // i18nPublished: string;
+  // i18nPublishLogUrlText: string;
+  // i18nPublishInProgress: string;
+  i18nUnpublish: string;
+  i18nUnpublishModalMessage: string;
+  i18nUnpublishModalTitle: string;
+  // icon?: string;
+  onDelete: (virtualizationName: string) => void;
+  /* TD-636: Commented out for TP
+  onExport: (virtualizationName: string) => void; */
+  onPublish: (virtualizationName: string, hasViews: boolean) => void;
+  onUnpublish: (virtualizationName: string) => void;
+  // publishingCurrentStep?: number;
```

Comment:
I'd prefer that the commented out properties be removed.  If we're sure they need to be added back later, then leave them but add a TODO

Reply:
Yeah.. agreed. Removed comments and added TODOs.